### PR TITLE
docs - value filter - list in/not-in/contains under comparison and list operators

### PR DESCRIPTION
**Pull request by ajkerrigan**
I noticed that we had `in`/`not-in` in the docs under "List Operators" and `contains` under "Comparison Operators". Those operators can all work against strings or lists though, so probably makes sense to list them in both locations.

We also briefly mention that `op: contains` is equivalent to `op: in` + `value_type: swap` but don't have an example to show it. May as well.

### CODE REVIEW

File: docs/source/filters.rst [hunk 1]
```diff
@@ -165,6 +169,15 @@ List Operators
            value: ID-321                  ─▶ Values to be compared against
            value_type: swap               ─▶ Switches list comparison order
 
+  The above filter can also be written as:
```
Comment (thisisshi) [inline docs/source/filters.rst:172]: maybe we should just remove the above example, it's much less intuitive after you realize `contains` exists

Comment (thisisshi) [reply docs/source/filters.rst:172]: or swap the two so that the contains example is first

Comment (ajkerrigan) [reply docs/source/filters.rst]: Wasn't sure of a way to clearly explain `op: difference` / `op: contains` / `value_type: swap` without pinning it to an example, so clustered example filters around a set of fake resources. Rendered preview from a local build:

image

**Review by kapilt — APPROVED** [review]
thanks, lgtm

**Pull request merged**